metabase/metabase
# Don't show "Download results" button until dashcard query is complete

Fixes #46019

Fixes dashcard menu had the "Download results" button while the card's query was still running

### To verify

1. Create a dashboard with at least one card
2. (Optionally) Enable network throttling
3. Open the dashboard page
4. Ensure you can't see the dashboard ellipsis menu until the query is complete

### Demo

https://github.com/user-attachments/assets/37230b79-2b5d-4488-a7c0-221a1f45ac4a
